duckduckgo/iOS
# Revert "Hung page detection (#1310)"

This reverts commit 4553af96fa922c0c42f17fef78ae839a0cc452f9.

Task/Issue URL: https://app.asana.com/0/414709148257752/1203449477057212/f
CC: @mallexxx 

**Steps to test this PR**:
1. Open bmxmuseum.com and browse forward/back.
2. Make sure app doesn't crash, doesn't reload unnecessarily and doesn't end up in a weird state.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
